classes/Forms.js

```diff
@@ -150,4 +151,9 @@ module.exports = class Forms extends OneBlinkAPI {
 
     return super.getRequest(`/forms/${formId}`)
   }
+
+  async validateForm (form /* : Form */) /* : Promise<Form> */ {
```

Comment:
I think its worth making this a static method so that developers don't have to create an instance of `Forms`.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static